src/kl_exchange.jl

```diff
@@ -16,6 +16,78 @@ function d_criterion(model_matrix;
         (1 / size(model_matrix, 2))
 end
 
+
+"""
+$(TYPEDSIGNATURES)
+
+Criterion of A-optimality.
+
+"""
+function a_criterion(model_matrix;
+                     tolerance = 1e-9)
+    M = model_matrix' * model_matrix + (I * tolerance)
+    C = cholesky(M, check = false)
+    if !issuccess(C) return 0 end
+    M⁻¹ = inv(C)
+    size(model_matrix, 2)/tr(M⁻¹)/size(model_matrix, 1)
+end
+
+"""
+$(TYPEDSIGNATURES)
+
+Criterion of T-optimality. Experimental.
```

Comment:
Add short descriptions on other criteria too, they go directly to the docs

Reply:
I didn't make test. This criterion maximize ``trace(X^T · X)``. I think there are discrepancies in the definition of T-optimalit in sources...